app/org/sagebionetworks/bridge/BridgeUtils.java

```diff
@@ -23,6 +36,75 @@
 
 public class BridgeUtils {
     
+    private static final Logger logger = LoggerFactory.getLogger(BridgeUtils.class);
+
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    /**
+     * Used for the Stormpath REST API. This should go away when they update their SDK.
+     * @param client
+     * @param url
+     * @param node
+     * @return
+     * @throws Exception
+     */
+    public static ObjectNode getJSON(CloseableHttpClient client, String url) throws Exception {
```

Comment:
Also, "throws Exception" is bad style. You should specify the specific exceptions you throw.